src/components/AddPlaidBankAccount.js

```diff
@@ -253,7 +253,7 @@ class AddPlaidBankAccount extends React.Component {
                                     label: this.props.translate('bankAccount.chooseAnAccount'),
                                 } : {}}
                                 value={this.state.selectedIndex}
-                                hasError={this.getErrors().selectedBank}
+                                errorText={this.getErrors().selectedBank}
```

Comment:
I don't think this is right. `getErrors().selectedBank` will return a boolean, but we want a translated error message here...

<img width="377" alt="Screen Shot 2022-05-04 at 3 37 46 PM" src="https://user-images.githubusercontent.com/22219519/166693206-447704d8-5eaa-441c-a971-e828e1a3ecf2.png">

<img width="377" alt="Screen Shot 2022-05-04 at 3 38 33 PM" src="https://user-images.githubusercontent.com/22219519/166693238-6d903624-a539-4792-af22-8e9b75ad4e02.png">

Reply:
I have a CORS error trying to get to `Connect banck account` page to tests my changes. My branch is updated with `main`.  Any suggestion? Also where `CompanyStep.js` is being used?

Reply:
Navigate to url  localhost:8080/bank-account/company

Reply:
Thank you!, I found the `Company Type` picker but I can't still find the `Account` Picker.  In what url I can find  `Account` Picker?

Reply:
You can reach it by navigating to `Settings > Workspace > Connect bank account > Connect online with Plaid`. [This thread](https://expensify.slack.com/archives/C01GTK53T8Q/p1646163503484199) has some info if you get into errors. TLDR you have to enable secure staging server.